tools/mbpp/fix_v011.py

```diff
@@ -0,0 +1,43 @@
+def check_id(data, n, task_id):
+    assert data[n]["task_id"] == task_id
+
+
+def fix(data):
+    # fix: https://github.com/evalplus/evalplus/issues/210
+
+    check_id(data, 215, "Mbpp/459")
+    data[215]["canonical_solution"] = data[215]["canonical_solution"].replace(
+        "if c.islower()", "if not c.isupper()"
+    )
+
+    return data
+
+
+if __name__ == "__main__":
+    import json
+
+    TASK_INSPECT = [
+        "Mbpp/459",
+    ]
+    SOURCE_VERSION = "v0.2.0"
+    TARGET_VERSION = "v0.3.0"
```

Comment:
`v0.2.1`

We only do middle version change when we add or remove some problems.